cmd/libs/go2idl/args/args.go

```diff
@@ -37,10 +37,12 @@ import (
 // Default returns a defaulted GeneratorArgs. You may change the defaults
 // before calling AddFlags.
 func Default() *GeneratorArgs {
-	return &GeneratorArgs{
+	generatorArgs := &GeneratorArgs{
 		OutputBase:       DefaultSourceTree(),
 		GoHeaderFilePath: filepath.Join(DefaultSourceTree(), "k8s.io/kubernetes/hack/boilerplate/boilerplate.go.txt"),
 	}
+	generatorArgs.AddFlags(pflag.CommandLine)
```

Comment:
On second thought, I think the argument can go either way. Users are supposed to call `Default()` to generate the args, and if they don't, probably they don't want to parse the flags either. So LGTM :)